gensrc/proto/segment.proto

```diff
@@ -167,6 +167,14 @@ message JsonMetaPB {
     optional bool has_remain = 3;
 
     optional bytes remain_filter = 4;
+
+    optional bool flat_json_enable = 5;
+
+    optional double flat_json_null_factor = 6;
+
+    optional double flat_json_sparsity_factory = 7;
+
+    optional int64 flat_json_column_max = 8;
```

Comment:
this code doesn't used？

Reply:
This is very important code, as it is used in json_flatten.cpp. For example:

if (column_meta_pb.json_meta().has_flat_json_null_factor()) {
    _max_json_null_factor = column_meta_pb.json_meta().flat_json_null_factor();
}
if (column_meta_pb.json_meta().has_flat_json_sparsity_factory()) {
    _min_json_sparsity_factory = column_meta_pb.json_meta().flat_json_sparsity_factory();
}
if (column_meta_pb.json_meta().has_flat_json_column_max()) {
    _max_column = column_meta_pb.json_meta().flat_json_column_max();
}